MAUI/Cartesian-Charts/Axis/Types.md

```diff
@@ -334,6 +335,192 @@ chart.XAxes.Add(primaryAxis);
 
 ![DateTimeAxis range customization support in MAUI Chart](Axis_Images/maui_chart_datetime_axis_range.jpg)
 
+## Logarithmic Axis
+
+The [LogarithmicAxis]() uses a logarithmic scale, and it is very useful in visualizing data when the data range of a chart’s series has significant disparity. It can be used either on the x-axis or the chart's y-axis. The Logarithmic Axis does not support zero or negative values. The following code example demonstrates how to set the logarithmic axis in the chart.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+
+    <chart:SfCartesianChart.XAxes>
+        <chart:LogarithmicAxis/>
+    </chart:SfCartesianChart.XAxes>
+    <chart:SfCartesianChart.YAxes>
+        <chart:LogarithmicAxis/>
+    </chart:SfCartesianChart.YAxes>
+
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+
+. . .
+
+LogarithmicAxis primaryAxis = new LogarithmicAxis();
+chart.XAxes.Add(primaryAxis);
+
+LogarithmicAxis secondaryAxis = new LogarithmicAxis();
+chart.YAxes.Add(secondaryAxis);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LogarithmicAxis support in MAUI Chart](Axis_Images/maui_chart_logarithmic_axis.jpg)
+
+### Customizing the range
+
+To customize the range of the log axis, you can use the [Minimum](), [Maximum]() and [Interval]() properties of the [LogarithmicAxis](). By default, an excellent range will be calculated automatically based on the provided data. The following code example demonstrates the range customization of the axis.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart.YAxes>
+
+    <chart:LogarithmicAxis Minimum="100" Maximum="10000"/>
+
+</chart:SfCartesianChart.YAxes>
+
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+chart.YAxes.Add(new LogarithmicAxis() { 
+
+	Minimum = 100, 
+	Maximum = 10000 
+
+});
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LogarithmicAxis range customization in MAUI Chart](Axis_Images/maui_chart_logarithmic_axis_range.jpg)
+
+### Logarithmic base
+
+The log axis base can be customized using the [LogarithmicBase]() property of the axis. By default, the base of a logarithmic axis is set to 10. You can, however, set the base to any value that meets your requirements. Just ensure that the base value is any positive number greater than 1 and cannot be a decimal number. The following code example demonstrates the logarithmic values in the y-axis calculated from base 2.
```

Comment:
The logarithmic axis base can be customized using the [LogarithmicBase]() property of the axis. By default, the base value is 10. The following code example demonstrates how to customize the base value. 

The above content is enough for base.

Reply:
@Saravanan-Madhesh , Content modified as requested.